### PR TITLE
Add form for Conditional

### PR DESCRIPTION
**Pull request by tnagorra**
## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/types/newAnalyticalFramework.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
 
 export type Types = WidgetTypes;
 
-type Conjunction = 'XOR' | 'OR' | 'AND' | 'NOR' | 'NAND' | 'NXOR';
+export type Conjunction = 'XOR' | 'OR' | 'AND';
```
Comment (samshara) [inline app/types/newAnalyticalFramework.ts:11]: type defined in ConjuctionInput component

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/OrganigramConditionalWidgetForm/DescendentOrganigramConditionInput/index.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <MultiSelectInput
+                name="value"
+                value={value.value}
+                onChange={onFieldChange}
+                error={arrayError}
+                options={valueOptions}
+                keySelector={optionKeySelector}
+                labelSelector={optionLabelSelector}
+            />
+        </>
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/OrganigramConditionalWidgetForm/DescendentOrganigramConditionInput/index.tsx]:
```suggestion
      <MultiSelectInput
            name="value"
            value={value.value}
            onChange={onFieldChange}
            error={arrayError}
            options={valueOptions}
            keySelector={optionKeySelector}
            labelSelector={optionLabelSelector}
        />
```

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/ScaleConditionalWidgetForm/ComparisonScaleConditionInput/index.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            <SelectInput
+                name="value"
+                value={value.value}
+                onChange={onFieldChange}
+                error={arrayError}
+                options={options}
+                keySelector={optionKeySelector}
+                labelSelector={optionLabelSelector}
+            />
+        </>
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/ScaleConditionalWidgetForm/ComparisonScaleConditionInput/index.tsx]:
```suggestion
        <SelectInput
            name="value"
            value={value.value}
            onChange={onFieldChange}
            error={arrayError}
            options={options}
            keySelector={optionKeySelector}
            labelSelector={optionLabelSelector}
        />
```

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/ScaleConditionalWidgetForm/SimpleScaleConditionInput/index.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            <MultiSelectInput
+                name="value"
+                value={value.value}
+                onChange={onFieldChange}
+                error={arrayError}
+                options={options}
+                keySelector={optionKeySelector}
+                labelSelector={optionLabelSelector}
+            />
+        </>
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/ScaleConditionalWidgetForm/SimpleScaleConditionInput/index.tsx]:
```suggestion
        <MultiSelectInput
            name="value"
            value={value.value}
            onChange={onFieldChange}
            error={arrayError}
            options={options}
            keySelector={optionKeySelector}
            labelSelector={optionLabelSelector}
        />
```

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/SingleSelectConditionalWidgetForm/SimpleSingleSelectConditionInput/index.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            <MultiSelectInput
+                name="value"
+                value={value.value}
+                onChange={onFieldChange}
+                error={arrayError}
+                options={options}
+                keySelector={optionKeySelector}
+                labelSelector={optionLabelSelector}
+            />
+        </>
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/SingleSelectConditionalWidgetForm/SimpleSingleSelectConditionInput/index.tsx]: remove unnecessary fragment code

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/TextConditionalWidgetForm/SimpleTextConditionInput/index.tsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    const error = getErrorObject(riskyError);
+
+    return (
+        <>
+            <TextInput
+                name="value"
+                value={value.value}
+                onChange={onFieldChange}
+                error={error?.value}
+            />
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/TextConditionalWidgetForm/SimpleTextConditionInput/index.tsx]: remove unnecessary fragment code

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/TimeRangeConditionalWidgetForm/SimpleTimeRangeConditionInput/index.tsx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+    return (
+        <>
+            <TimeInput
+                name="value"
+                value={value.value}
+                onChange={onFieldChange}
+                error={error?.value}
+            />
+        </>
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetConditionalEditor/TimeRangeConditionalWidgetForm/SimpleTimeRangeConditionInput/index.tsx]: remove unnecessary fragment

File: app/types/newAnalyticalFramework.ts (hunk 1 above)
Comment (tnagorra) [reply app/types/newAnalyticalFramework.ts:11]: IMO we can duplicate simple types, better to avoid coupling when possible.

**Pull request merged**